packages/components/docs/token-table.module.css

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
I'll set up stylelint with prettier and it should take care of this for us (though I personally kind of like a leading newline) helps me see it more easily since it gives distance from the tabs above.